src/services/business-lookup-services.ts

```diff
@@ -28,14 +28,14 @@ export default class BusinessLookupServices {
   }
 
   /**
-   * Searches for code or words.
-   * @param query words to search
+   * Searches for business by code or words.
+   * @param query code or words to search
    * @returns a promise to return the search results
    */
   static async search (query: string): Promise<BusinessLookupResultIF[]> {
-    const legalType = 'BC' // Will be updating to a list once search api support it.
+    const legalType = 'BC,A,ULC,C,S,XP,GP,LP,CUL,XS,LLC,LL,BEN,CP,SP,CC,XL,FI,XCP,PA'
```

Comment:
I would prefer alphabetical, but this is the order provided by the business so this makes it easy to compare and verify the list.